### ISSUE TITLE
Laravel 5.1

### ISSUE THREAD

**Issue by eduardocintra** [limweb/L5MPDF#6]
I know, that this is not a issue!
But I don't know where get help.

When I try to install the mpdf in laravel 5.1 I get the following error:

 [InvalidArgumentException]  
  Could not find package servit/l5mpdf at any version for your minimum-stabil  
  ity (stable). Check the package spelling or your minimum-stability 

Does anyone have any idea how to fix?

**Comment by niklasravnsborg** [comment]
@eduardocintra Maybe try including following two lines in composer.json:

```
"mpdf/mpdf": "dev-master@dev",
"servit/l5mpdf": "dev-master@dev"
```

then run `composer update`.
Hope that helps 😌

**Comment by HawiCaesar** [comment]
![Uploading Screenshot from 2016-06-07 00:17:59.png…]()
even with that, I cant get it to run I get error 500 when I try and follow your guide on the usage

**Comment by niklasravnsborg** [comment]
@HawiCaesar Yes composer won't allow unstable development versions by default. Sorry for the self promotion here but I also kept struggling with that so I created an own PDF package which is also based on MPDF: https://github.com/niklasravnsborg/laravel-pdf

Maybe this will help you move forward...
